### PR TITLE
update to ember and ember-cli 2.17.0

### PR DESCRIPTION
**Pull request by miguelcobain**
Lots of cools stuff in this update.
Credit goes mostly to the awesome ember-cli-update package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ This background-color style will only be applied if `image` is a "falsy" value:
 
 ## Installation
 
-* `git clone` this repository
+* `git clone <repository-url>` this repository
+* `cd my-addon`
```
Comment (k-fish) [inline README.md]: `cd my-addon`?

**Comment by k-fish** [comment]
Aside from the readme typo, looks good.

**Comment by miguelcobain** [comment]
@k-fish typo was fixed

**Comment by k-fish** [comment]
Nice, thanks @miguelcobain

**Comment by miguelcobain** [comment]
@k-fish no problem. :)
it would be nice if you could cut a new release.

**Comment by k-fish** [comment]
@miguelcobain, tagged and published. Give it a shot.

**Pull request merged**